content/docs/2.1/concepts/authentication.md

```diff
@@ -131,6 +132,33 @@ Every parameter you define in `TriggerAuthentication` definition does not need t
       name: {trigger-authencation-name} # this may define other params not defined in metadata
 ```
 
+## Authentication scopes: Namespace vs. Cluster
+
+Each `TriggerAuthentication` is defined in one namespace and can only be used by a `ScaledObject` in that same namespace. For cases where you want to share a single set of credentials between scalers in many namespaces, you can instead create a `ClusterTriggerAuthentication`. As a global object, this can be used from any namespace. To set a trigger to use a `ClusterTriggerAuthentication`, add a `kind` field to the authentication reference:
+
+```yaml
+    authenticationRef:
+      name: {trigger-authencation-name}
+      kind: ClusterTriggerAuthentication
+```
+
+By default, Secrets loaded from a `secretTargetRef` must be in the same namespace as KEDA is deployed in (usually `keda`). This can be overridden by setting a `$KEDA_CLUSTER_OBJECT_NAMESPACE` environment variable for the `keda-operator` container.
```

Comment:
Does it really need the `$` in the environment variable?
```suggestion
By default, Secrets loaded from a `secretTargetRef` must be in the same namespace as KEDA is deployed in (usually `keda`). This can be overridden by setting a `KEDA_CLUSTER_OBJECT_NAMESPACE` environment variable for the `keda-operator` container.
```

Reply:
I usually put it as a secondary reminder than it's an env var. That can backfire if people take it literally though.